script/GenKey.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# usage: ./GenKey.sh [<directory_name>]
+
+# get blpapi-http root directory
+DNAME=${1:-temp}
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && mkdir "$DNAME" && cd "$DNAME" && pwd )
+
+# generate ca private key
+openssl genrsa -out "$DIR"/ca-key.pem 1024
+# generate private key for server
+openssl genrsa -out "$DIR"/server-key.pem 1024
+# generate private key for server
+openssl genrsa -out "$DIR"/client-key.pem 1024
+
+# generate ca certs
+openssl req -new -x509 -text -key "$DIR"/ca-key.pem \
+    -out "$DIR"/ca-cert.pem -subj "/CN=BLPAPI_HTTP_TEST_CA/"
+
+# generate server cert
+openssl req -new -key "$DIR"/server-key.pem \
+    -out "$DIR"/server-csr.pem -text -subj "/CN=localhost/"
+openssl x509 -req -in "$DIR"/server-csr.pem \
+    -CA "$DIR"/ca-cert.pem -CAkey "$DIR"/ca-key.pem \
+    -set_serial 01 -out "$DIR"/server-cert.pem -text
+
+# generate client cert
+openssl req -new -key "$DIR"/client-key.pem \
+    -out "$DIR"/client-csr.pem -text -subj "/CN=BLPAPI_HTTP_TEST_CLIENT/"
+openssl x509 -req -in "$DIR"/client-csr.pem -extensions usr_cert \
```

Comment:
I tries to add some `X509v3 extensions` elements to the issued certificate. I missed it while testing the ssl issue unintentionally. Will remove it later.